pkg/webhooks/annotations.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 const (
-	policyAnnotation = "policies.kyverno.io~1patches"
+	policyAnnotation = "policies.kyverno.io~1last-applied-patches"
```

Comment:
Will the old annotation be removed?

Reply:
I wasn't sure how to go about this earlier, I will work on this with Pooja now.